# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/abiword/hotfix-libjpeg.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- abiword-3.0.5/src/af/util/xp/ut_jpeg.cpp.vanilla	2024-05-08 19:24:02.933000000 +0200
+++ abiword-3.0.5/src/af/util/xp/ut_jpeg.cpp	2024-05-08 19:26:29.101000000 +0200
@@ -102,7 +102,7 @@
 	src->pub.next_input_byte = src->sourceBuf->getPointer (src->pos);
 	src->pub.bytes_in_buffer = src->sourceBuf->getLength ();
 
-	return 1; // boolean is a libjpeg type that is an int.
+	return (boolean)1; // boolean is a libjpeg type that is an int.
 }
 
 /*
